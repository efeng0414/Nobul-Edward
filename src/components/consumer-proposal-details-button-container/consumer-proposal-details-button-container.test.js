import React from "react";
import renderer from "react-test-renderer";
import { IntlProvider } from "react-intl";

import ConsumerProposalDetailsButtonContainer from "./index";

describe("ConsumerProposalDetailsButtonContainer", () => {
  const componentProps = {
    intl: global.getIntl(),
    handleProposalAcceptClick: () => {},
    handleProposalDeclineClick: () => {}
  };

  const componentJSX = (
    <ConsumerProposalDetailsButtonContainer {...componentProps} />
  );

  describe("Component Snapshot", () => {
    const snapshotJSX = (
      <IntlProvider {...global.intl}>{componentJSX}</IntlProvider>
    );

    const componentSnapshot = renderer.create(snapshotJSX).toJSON();
    it("should matched stored snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
