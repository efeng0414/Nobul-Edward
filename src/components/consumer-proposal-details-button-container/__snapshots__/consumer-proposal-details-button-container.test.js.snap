// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConsumerProposalDetailsButtonContainer Component Snapshot should matched stored snapshot 1`] = `
<div
  className="accept-decline-buttons"
>
  <div
    className="accept-decline-buttons__accept"
  >
    <button
      className="ant-btn decline-button ant-btn-lg"
      onClick={[Function]}
      type="button"
    >
      <span>
        Decline
      </span>
    </button>
  </div>
  <div
    className="accept-decline-buttons__decline"
  >
    <button
      className="ant-btn accept-button ant-btn-primary ant-btn-lg"
      onClick={[Function]}
      type="button"
    >
      <span>
        Accept
      </span>
    </button>
  </div>
</div>
`;
